CONTRIBUTING.md

```diff
@@ -0,0 +1,171 @@
+# How to contribute to Drägerwerk sdc11073
+
+Welcome to our project. We are glad you want to improve this open source library. We at Dräger want to create technology
+for life and help to create a better future 🚀.
+
+If you provide Markdown documents, issues, pull request or discussions we strongly encourage you
+to [format your text](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
+as it greatly helps to read and to understand your provided information.
+
+For this open source project
+the [Contributor Licence Agreement](https://github.com/Draegerwerk/sdc11073/Contributor_License_Agreement.md) governs
```

Comment:
It appears this link would not work. I tried to use the same kind of link with another file name of a file that already exists on the master branch and it did not work.

Perhaps it should be changed according to the following suggestion?

```suggestion
the [Contributor Licence Agreement](https://github.com/Draegerwerk/sdc11073/blob/master/Contributor_License_Agreement.md) governs
```